packages/audit_tracker/Gemfile.lock

```diff
@@ -186,4 +191,4 @@ DEPENDENCIES
   sqlite3 (~> 1.4.2)
 
 BUNDLED WITH
-   2.3.26
```

Comment:
@xjunior we should add the appraisal stuff to our workflow. CI did not catch that this wasn't up-to-date with the version of bundler I bumped in the last PR

Reply:
How would appraisal help with this?

Reply:
wait....why is audit_tracker in here? I added it https://github.com/powerhome/power-tools/pull/226/files#diff-ba18d7e10ae7c64e586ed183e8dd74117e08b93c4bbabe05052a6d516a0e8722R194

Reply:
I was not paying attention and thought this was an apparaisal gemfile. But apparently it was just the fact that I hadn't pulled `main` before I opened this branch. 🤷‍♀️ not sure why github was showing it in the diff, since there's no different from main. But pulling main made it go away